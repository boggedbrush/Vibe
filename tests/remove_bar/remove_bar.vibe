# VibeSpec: 1.6

# Remove the `bar` function from hello.py
patch_type: remove_function
file: hello.py
name: bar
