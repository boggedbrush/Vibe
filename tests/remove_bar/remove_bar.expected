def foo(x):
    print(f'foo({x})')


class MyClass:

    def compute(self, value):
        pass

    def old_method(self, *args):
        print('MyClass.old_method', *args)
